docs_src/microservices/support/scheduler/GettingStarted.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Support Scheduler - Getting Started
+---
+
+# Support Scheduler - Getting Started
+
+Support Scheduler is one of the core EdgeX Services. It is needed for applications that require actions to occur on specific intervals.
+For solutions that do not require regular actions, it is possible to use the EdgeX framework without support scheduler.
+
+## Running Services with Support Notifications
+
+The simplest way to run all the required services is to use the [Compose Builder](https://github.com/edgexfoundry/edgex-compose/tree/{{edgexversion}}/compose-builder) tool from a terminal window.
+
+1. Clone [https://github.com/edgexfoundry/edgex-compose/tree/{{edgexversion}}](https://github.com/edgexfoundry/edgex-compose/tree/{{edgexversion}})
+
+2. Change to the **compose-builder** folder
+
+3. Run the services
+    ```
+    make run no-secty
+    ```
+This runs, in non-secure mode, all the standard EdgeX services, including support scheduler, along with the Device Virtual.
+
+## Running Services without Core Data
+The simplest way to run all the required services is to use the [Compose Builder](https://github.com/edgexfoundry/edgex-compose/tree/{{edgexversion}}/compose-builder) tool from a terminal window.
+
+1. Clone [https://github.com/edgexfoundry/edgex-compose/tree/{{edgexversion}}](https://github.com/edgexfoundry/edgex-compose/tree/{{edgexversion}})
+2. Change to the **compose-builder** folder
+3. Generate a compose file
+    ```
+    make gen no-secty
+    ```
+4. Remove support-scheduler from the compose file and resolve any depends on for support scheduler.
+5. Run the compose file.
+    ```
+    make up
+    ```
+This runs, in non-secure mode, all the standard EdgeX services, except for support scheduler, along with the Device Virtual.
```

Comment:
Needs `ds-virtual` option for Device Virtual to be running

Reply:
don't need `ds-virtual`
```
make run no-secty
docker compose -p edgex -f docker-compose-no-secty.yml up -d 
[+] Running 19/19
 ✔ Network edgex_edgex-network                      Created                                                                           0.0s 
 ✔ Volume "edgex_kuiper-log"                        Created                                                                           0.0s 
 ✔ Volume "edgex_kuiper-plugins"                    Created                                                                           0.0s 
 ✔ Volume "edgex_db-data"                           Created                                                                           0.0s 
 ✔ Volume "edgex_kuiper-data"                       Created                                                                           0.0s 
 ✔ Volume "edgex_kuiper-etc"                        Created                                                                           0.0s 
 ✔ Container edgex-core-consul                      Started                                                                           0.8s 
 ✔ Container edgex-redis                            Started                                                                           0.9s 
 ✔ Container edgex-ui-go                            Started                                                                           0.6s 
 ✔ Container edgex-core-common-config-bootstrapper  Started                                                                           1.1s 
 ✔ Container edgex-support-notifications            Started                                                                           1.3s 
 ✔ Container edgex-core-metadata                    Started                                                                           1.5s 
 ✔ Container edgex-kuiper                           Started                                                                           1.5s 
 ✔ Container edgex-support-scheduler                Started                                                                           1.2s 
 ✔ Container edgex-core-data                        Started                                                                           1.9s 
 ✔ Container edgex-core-command                     Started                                                                           1.9s 
 ✔ Container edgex-device-virtual                   Started                                                                           2.3s 
 ✔ Container edgex-device-rest                      Started                                                                           2.5s 
 ✔ Container edgex-app-rules-engine                 Started   
 ```